l10n_it_invoices_data_communication/models/account_invoice.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+
+from openerp import fields, models, _
+from openerp.exceptions import ValidationError, Warning as UserError
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    comunicazione_dati_iva_escludi = fields.Boolean(
+        string='Exclude from invoices communication')
+
+    def _compute_taxes_in_company_currency(self, vals):
+        sign = 1 if self.type in ('out_invoice', 'in_refund') else -1
+        amount_total_signed = sign * self.amount_total
+        currency_id = self.currency_id.with_context(date=self.date_invoice)
+        amount_total_company = currency_id.compute(
+            self.amount_total, self.company_id.currency_id)
+        amount_total_company_signed = sign * amount_total_company
+        try:
+            exchange_rate = (
+                amount_total_signed /
+                amount_total_company_signed)
+        except ZeroDivisionError:
+            exchange_rate = 1
+        vals['ImponibileImporto'] = vals['ImponibileImporto'] / exchange_rate
+        vals['Imposta'] = vals['Imposta'] / exchange_rate
+
+    def _get_tax_comunicazione_dati_iva(self):
+        self.ensure_one()
+        fattura = self
+        tax_model = self.env['account.tax']
+
+        tax_lines = []
+        tax_grouped = {}
+        for tax_line in fattura.tax_line:
+            tax = tax_model.search([
+                ('tax_code_id', '=', tax_line.tax_code_id.id)
+            ], limit=1)
+            if not tax:
+                raise UserError(
+                    _("Tax with code {tax_code} not found")
+                    .format(
+                        tax_code=tax_line.tax_code_id.display_name))
+            aliquota = tax.amount
```

Comment:
nella 8.0 è in %, quindi va moltiplicata * 100

Reply:
Risolto, insieme ad altre occorrenze di `account.tax.amount`